packages/kolme/src/submitter/solana.rs

```diff
@@ -72,15 +72,17 @@ pub async fn execute(
     };
 
     let blockhash = client.get_latest_blockhash().await?;
+    tracing::info!("Constructing and sending tx using blockhash {blockhash}");
     let tx =
         signed_tx(program_id, blockhash, keypair, &data, &metas).map_err(|x| anyhow::anyhow!(x))?;
 
     match client.send_and_confirm_transaction(&tx).await {
         Ok(sig) => Ok(sig.to_string()),
         Err(e) => {
             tracing::error!(
-                "Solana submitter failed to execute signed transaction: {}",
-                e
+                "Solana submitter failed to execute signed transaction: {}, error kind: {:?}",
```

Comment:
Should `e` also be using `{:?}` ?

Reply:
The problem is that Display for this error uses Display of its field `kind` but it uses `#[error(transparent)]` hiding the actual kind of the error. Also that error includes `Option<request::RpcRequest>` but I don't think we need this in traces.